VTEX - Logistics API.json

```diff
@@ -6962,8 +6954,8 @@
                 "tags": [
                     "Pickup points"
                 ],
-                "summary": "List all pickup points",
-                "description": "Lists all of your store's pickup points.",
+                "summary": "List pickup points",
+                "description": "Retrieves information about [pickup points](https://help.vtex.com/en/tutorial/pickup-points--2fljn6wLjn8M4lJHA6HP3R) of your store.\r\n\r\n>⚠️ The response is limited to 1.000 pickup points.",
```

Comment:
I think the description should mention the endpoint that returns a paged list as an alternative if you need more than 1000 results.

Reply:
Good ideia.